tests/Config/InfectionConfigTest.php

```diff
@@ -52,6 +52,17 @@ public function test_it_returns_default_phpunit_config_dir_with_no_config()
         $this->assertSame('/path/to/config', $config->getPhpUnitConfigDir());
     }
 
+    public function test_it_returns_phpunit_absolute_dir_from_config_with_absolute_path()
+    {
+        $absolutePath = '/app';
+        $json = sprintf('{"phpUnit": {"configDir": "%s"}}', $absolutePath);
+        $config = new InfectionConfig(json_decode($json), $this->filesystem, '/path/to/config');
+
+        $expected = $absolutePath;
```

Comment:
could you please explain what is the idea behind this?

Reply:
agree, seems redundant. @adeptofvoltron could you please remove `$expected` and use `$absolutePath` or explain why it is needed?
Then I guess it will be ready to merge